### PR TITLE
Add the Scroll Concierge™.

### PR DESCRIPTION
**Pull request by DFurnes**
#### Changes
This pull request includes the __Scroll Concierge™__, which can guide the user to a particular area on the page upon load. It also includes a small fix to `FlexCell` prop types & an incorrect docblock.

I'm currently figuring out some occasional bugs, so not ready to merge yet. 🚧

### CODE REVIEW

File: resources/assets/components/ContentPage/index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import Markdown from '../Markdown';
 import CallToActionContainer from '../../containers/CallToActionContainer';
+import ScrollConcierge from '../ScrollConcierge';
```
Comment (weerd) [inline resources/assets/components/ContentPage/index.js]: yais! I'm glad we're sticking w/ this name!

File: resources/assets/components/ScrollConcierge/index.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      clearInterval(scrollInterval);
+    }
+
+    window.scrollBy(0, scrollStep);
+  }, 15);
+};
+
+class ScrollConcierge extends React.Component {
+  componentDidMount() {
+    setTimeout(() => scrollTo(this.node.offsetTop - 150), 0);
```
Comment (deadlybutter) [inline resources/assets/components/ScrollConcierge/index.js]: What is this 150px? Maybe add a comment

**Comment by DFurnes** [comment]
Hey hey! Pushed up the final version of this. Changes include:

 - Switched to using `requestAnimationFrame` for less janky animations.
 - Switched the Lede Banner to use `display: table-cell` for layout rather than Flexbox, since Flexbox [renders a little weirdly](https://jakearchibald.com/2014/dont-use-flexbox-for-page-layout/) and that would cause the offset of the top of the "page" to shift by the time we'd scrolled there.
- Added some fallback fonts for League Gothic to make it less likely we use a big ol' sans-serif that doesn't really look at all like the final font.
- Adds [FontFaceObserver](https://github.com/bramstein/fontfaceobserver) to wait until League Gothic has loaded before we calculate the scroll offset (since otherwise we'd scroll too far – see [without League Gothic](https://cloud.githubusercontent.com/assets/583202/26316348/f42dc528-3ee1-11e7-9332-606a1528356e.png) and [with League Gothic](https://cloud.githubusercontent.com/assets/583202/26316349/f68a2a8c-3ee1-11e7-9adb-783fb459287d.png)).

**Comment by DFurnes** [comment]
For your viewing pleasure, here's the [FAQ page](https://phoenix-next-stage-pr-266.herokuapp.com/us/campaigns/sincerely-us/pages/faqs) with the concierge on it.

File: resources/assets/components/LedeBanner/lede-banner.scss [hunk 3]
```diff
@@ -51,8 +51,9 @@
 
 .lede-banner__content {
   @include media($medium) {
-    flex: 1;
+    display: table-cell;
```
Comment (weerd) [inline resources/assets/components/LedeBanner/lede-banner.scss]: oldie but goodie 😉

**Comment by DFurnes** [comment]
[(5) Page Concierge for the Scholarship Page](https://trello.com/c/hzFrCqBp/635-5-page-concierge-for-the-scholarship-page)

**Pull request merged**